app/app.go

```diff
@@ -483,16 +453,6 @@ func NewWasmApp(
 		scopedIBCKeeper,
 	)
 
-	// Register the proposal types
-	// Deprecated: Avoid adding new handlers, instead use the new proposal flow
-	// by granting the governance module the right to execute the message.
-	// See: https://docs.cosmos.network/main/modules/gov#proposal-messages
-	govRouter := govv1beta1.NewRouter()
```

Comment:
The legacy router is still used for some modules. See https://github.com/cosmos/cosmos-sdk/blob/v0.47.4/simapp/app.go#L357-L360